src/Form/InvoiceType.php

```diff
@@ -22,11 +23,21 @@ public function buildForm(FormBuilderInterface $builder, array $options): void
         $builder
             ->add('name')
             ->add('description')
-            ->add('typePayment')
+            ->add('typePayment', ChoiceType::class, [
+                'attr' => [
+                    'placeholder' => 'Choisissez le type de payment'
+                ],
+                'choices' => [
+                    'Carte' => 'Carte',
+                    'Chèque' => 'Chèque',
+                    'Espèce' => 'Espèce',
```

Comment:
Is the key, the value that we will stock in database ? if it's the case, we should name it in english since we have everything in english into the database

Reply:
et c'est pas un truc qui vient de la db c'est juste du texte que tu vas ajouter

Reply:
La donnée qu'on récupère vient d'ici ?

Reply:
oui dans l'imput du form

Reply:
Dans ce cas la valeur qui est récupérée est en francais sauf que tout est en anglais dans la base de données